freesound/urls.py

```diff
@@ -66,6 +66,7 @@
     url(r'^people/(?P<username>[^//]+)/packs/(?P<pack_id>\d+)/delete/$', sounds.views.pack_delete, name="pack-delete"),
     url(r'^people/(?P<username>[^//]+)/packs/(?P<pack_id>\d+)/download/.*$', sounds.views.pack_download, name="pack-download"),
     url(r'^people/(?P<username>[^//]+)/packs/(?P<pack_id>\d+)/downloaders/$', sounds.views.pack_downloaders, name="pack-downloaders"),
+    url(r'^secret_retrieve/people/(?P<username>[^//]+)/packs/(?P<pack_id>\d+)/licenses/$', sounds.views.pack_licenses, name="pack-licenses"),
```

Comment:
I don't think this needs to have a `secret_retrieve` prefix. The `/secret` prefix for sounds is related to our use of `X-Accel-Redirect`, which maps to a file on disk. This URL is served from django, so I'm happy for it to be 

    /people/x/packs/y/license